interfaces/builtin/basedeclaration.go

```diff
@@ -209,6 +209,11 @@ slots:
       slot-snap-type:
         - core
     deny-auto-connection: true
+  connectivity-observe:
+    allow-installation:
+      slot-snap-type:
+        - app
+        - core
```

Comment:
This should also have:
```
  deny-connection: true
```

so that a snap declaration is required for the slot implementation to have plugging snaps connect to it. We want to allow auto-connection once the snap declaration is granted, so leaving that out is fine.